### PR TITLE
DATAUP-550 show errors missing or invalid values

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

This brings in most of the existing app cell functionality for showing missing or invalid value errors. 

For errors (missing-required, invalid values), these show up as red boxes beneath the input with an error prompt. They also color the input label and border red.

For warnings (currently just duplicates of existing data objects, not other outputs in the same run), these show up as a yellow box.

Screen Shot 2021-08-06 at 5 35 56 PM

The select dropdowns not being colored red (aside from their labels) is an issue with the Select2 box and will require separate work to fix. The "Reverse/Right FASTQ File Path" not being tagged as an error is Working As Intended - both of those input fields are set to optional in the spec: [here](https://github.com/kbaseapps/kb_uploadmethods/blob/7db3c27e23dbbbade89ea77c738c3ce33ac25c37/ui/narrative/methods/import_fastq_noninterleaved_as_reads_from_staging/spec.json). We might want to make a ticket out of that.

Tests are still pending, so maybe don't merge quite yet, but I wanted to get something up for discussion, rather than spamming Slack with screenshots.

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-550
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

**Comment by MrCreosote** [comment]
> The "Reverse/Right FASTQ File Path" not being tagged as an error is Working As Intended - both of those input fields are set to optional in the spec: here. We might want to make a ticket out of that.

That's because the non-interleaved app supports both single and and paired end reads, so the reverse read field has to be optional. Related ticket: https://kbase-jira.atlassian.net/browse/DATAUP-482

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 1]
```diff
@@ -74,7 +74,7 @@ $tab-widget-padding: 4px;
       margin: -($tab-widget-margin + $tab-widget-padding);
     }
 
-    // kb-bulk-import__filetype_panel
+    // kb-bulk-import-configure__filetype_panel
```
Comment (briehl) [inline kbase-extension/scss/partials/_bulkImportCell.scss:77]: These all had a little typo.

**Comment by MrCreosote** [comment]
https://github.com/kbaseapps/kb_uploadmethods/pull/331

File: kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js [hunk 2]
```diff
@@ -10,7 +10,24 @@ define([
     const t = html.tag,
         div = t('div'),
         label = t('label'),
-        cssBaseClass = 'kb-field-cell';
+        strong = t('strong'),
+        span = t('span'),
+        iTag = t('i'),
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js:15]: 🤞  this hasn't already been patented by Apple...

**Review by ialarmedalien — APPROVED** [review]
Looks great!

**Comment by briehl** [comment]
I'll go ahead and merge to get it up on n-r. I'll add some deeper unit tests to the next PR.

**Pull request merged**